src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -163,7 +163,7 @@
     <value>Section '{0}' is already defined.</value>
   </data>
   <data name="SectionAlreadyRendered" xml:space="preserve">
-    <value>The section named '{0}' has already been rendered.</value>
+    <value>The section named '{0}' has already been rendered. (Path: '{1}')</value>
```

Comment:
suggest `'{0}' invocation in '{2}' is invalid. Section '{1} has already been rendered.` where caller passes `nameof(RenderSectionAsync)` for the 0th argument.